core/src/main/java/org/apache/accumulo/core/file/streams/BoundedRangeFileInputStream.java

```diff
@@ -14,24 +14,22 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-
-package org.apache.accumulo.core.file.rfile.bcfile;
+package org.apache.accumulo.core.file.streams;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.security.AccessController;
 import java.security.PrivilegedActionException;
 import java.security.PrivilegedExceptionAction;
 
-import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.Seekable;
```

Comment:
Seekable is public/evolving which means it's probably OK for us to expect to be stable, but mentioning to make sure we consciously acknowledge that.